tools/ci_connector_ops/ci_connector_ops/qa_engine/enrichments.py

```diff
@@ -33,10 +38,13 @@ def get_enriched_catalog(oss_catalog: pd.DataFrame, cloud_catalog: pd.DataFrame)
     enriched_catalog.columns = enriched_catalog.columns.str.replace(
         "(?<=[a-z])(?=[A-Z])", "_", regex=True
     ).str.lower() # column names to snake case
+    enriched_catalog = enriched_catalog[[c for c in enriched_catalog.columns if "_del" not in c]]
```

Comment:
Optional: I took a minute to understand why we were removing these - maybe using the suffix `_cloud` would make it clear that when merging the two, we prefer the OSS column and drop the cloud one